app/views/purchases/_purchase.html.erb

```diff
@@ -9,19 +9,20 @@
   </div>
 
   <!-- Purchase Details -->
-  <div class="row mt-4">
+  <div class="row mt-4 bg-white">
     <div class="col-md-4">
-      <p>
```

Comment:
- [Optional] Dear @AbdelazizSoliman, Please Implement some validation on your forms to handle some exceptions that would arise on runtime. Currently, I keep getting an error while trying to submit a form without filling some forms. 

- [Optional] Dear @AbdelazizSoliman, For the design, Please consider following the guidelines closely. For the signout button, I would suggest using a color that is closely complemented with the Other colors on the page. Remember, as frontend engineers we are expected to deliver something identical. 

### Type of error.
![Screen Shot 2023-09-29 at 8 26 03 PM](https://github.com/AbdelazizSoliman/Budget_app/assets/69077061/5eb5312c-3b1b-4569-8af3-f5005e5aa91f)